### DISCUSSION TITLE
blog/immersive-translate/

### DISCUSSION THREAD

**Discussion by giscus** [sqybi/SQYBI.com#11] [Announcements]
# blog/immersive-translate/

最近状态不好，挑一些简单的 topic 来写吧。

https://sqybi.com/blog/immersive-translate/

**Comment by xuejc1988** [comment]
我去试试看。
我的企业微信工作签名是：一起投20面的骰子吧！
我的钉钉工作签名是：你看到了薛健聪，你觉得他看起来不是很耐烦
我的飞书工作签名是：WHOOP WHOOP,PULL UP,PULL UP
